File: Example4/app/src/main/java/com/example/example4/MainActivity.java
package com.example.example4;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    ImageView imageView ;
    TextView textView ;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

//        imageView = findViewById(R.id.imageView);
        textView = findViewById(R.id.textView);


    }
}

File: KotlinExample/app/src/main/java/com/example/kotlinexample/MainActivity.kt
package com.example.kotlinexample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val list = ArrayList<CModels>()
        list.add(CModels("googiehost","UsernameAndPassword"))
        list.add(CModels("googiehost","UsernameAndPassword"))
        list.add(CModels("googiehost","UsernameAndPassword"))
        list.add(CModels("googiehost","UsernameAndPassword"))
        list.add(CModels("googiehost","UsernameAndPassword"))
        var adapter:CAdapter? = null
        adapter = CAdapter(this,list)
        recycler.layoutManager = LinearLayoutManager(this)
        recycler.adapter = adapter

    }
}